test/object-store/util/sync/baas_admin_api.cpp

```diff
@@ -602,13 +609,19 @@ static std::optional<sync::RedirectingHttpServer>& get_redirector(const std::str
         });
     };
 
-    if (redirector_enabled() && !redirector && !base_url.empty()) {
+    if (!base_url.empty() && !redirector && redirector_enabled()) {
```

Comment:
Swapped the comparison arguments to hopefully be more efficient (i.e. only query the environment var if a base_url was provided and the redirector is empty).

Reply:
No longer needed.